test/advection_test/test_host.F90

```diff
@@ -223,8 +222,9 @@ subroutine test_host(retval, test_suites)
        use test_host_mod,      only: init_data, compare_data
        use test_host_mod,      only: ncols, pver
        use test_host_ccpp_cap, only: test_host_ccpp_register_constituents
+       use test_host_ccpp_cap, only: test_host_ccpp_initialize_constituents
        use test_host_ccpp_cap, only: test_host_ccpp_number_constituents
-       use test_host_ccpp_cap, only: test_host_advected_constituents
+       use test_host_ccpp_cap, only: test_host_constituents_array
```

Comment:
Changed it to the test_host_constituents_array because that more accurately reflects how we're using the constituent array (though the two arrays are identical)